packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_model.dart

```diff
@@ -163,13 +168,122 @@ class IosDomainError extends DomainError {
     'Ensure your domain is accessible without any redirects.',
   );
 
-  /// TODO(hangyujin): There are sub checkes of this check, add them and add links when finalized.
-  /// AASA file format follows guidelines.
-  static const fileFormat = IosDomainError(
-    'Apple-App-Site-Association file format is incorrect',
-    'This test checks that your Apple-App-Site-Association file '
-        'follows the correct format guidelines.',
-    'Ensure your Apple-App-Site-Association file follows the correct format guidelines.',
+  final List<AASAfileFormatSubCheck> subcheckErrors;
+}
+
+IosDomainError iosFileFormatDomainError({
```

Comment:
can this be a static method on IosDomainError for consistency with how the other errors are defined?

Reply:
updated, thanks!